howto/regex.po

```diff
@@ -32,160 +32,250 @@ msgstr ""
 msgid ""
-"We'll start by learning about the simplest possible regular expressions.  "
-"Since regular expressions are used to operate on strings, we'll begin with "
-"the most common task: matching characters."
+"We'll start by learning about the simplest possible regular expressions.  Since "
+"regular expressions are used to operate on strings, we'll begin with the most "
+"common task: matching characters."
 msgstr ""
+"Nous commencerons par étudier les expressions régulières les plus simples. Dans "
+"le mesure où les expressions régulières sont utilisées pour manipuler des "
+"chaînes de caractères, nous commencerons par les tâches les plus communes: "
+"caractères ressemblants."
 
 #: ../Doc/howto/regex.rst:57
 msgid ""
 "For a detailed explanation of the computer science underlying regular "
 "expressions (deterministic and non-deterministic finite automata), you can "
 "refer to almost any textbook on writing compilers."
 msgstr ""
+"Pour une explication détaillée sur la technologie sous-jacente des expressions "
+"régulières (automate d'état déterministe et non-déterministe), faites référence "
+"à n'importe quel manuel sur l'écriture de compilateurs."
 
 #: ../Doc/howto/regex.rst:63
 msgid "Matching Characters"
-msgstr ""
+msgstr "Caractères correspondants"
 
 #: ../Doc/howto/regex.rst:65
 msgid ""
 "Most letters and characters will simply match themselves.  For example, the "
-"regular expression ``test`` will match the string ``test`` exactly.  (You "
-"can enable a case-insensitive mode that would let this RE match ``Test`` or "
+"regular expression ``test`` will match the string ``test`` exactly.  (You can "
+"enable a case-insensitive mode that would let this RE match ``Test`` or "
 "``TEST`` as well; more about this later.)"
 msgstr ""
+"La plupart des lettres ou caractères vont correspondre à eux-mêmes . Par "
+"exemple, l'expression régulière ``test`` correspond à la chaîne de caractère "
+"``test`` précisément. (Vous pouvez activer le mode non-sensible à la casse qui "
+"permet à cette RE de correspondre à ``Test`` ou ``TEST`` également; plus à ce "
+"sujet dans la suite.) "
 
 #: ../Doc/howto/regex.rst:70
 msgid ""
 "There are exceptions to this rule; some characters are special :dfn:"
-"`metacharacters`, and don't match themselves.  Instead, they signal that "
-"some out-of-the-ordinary thing should be matched, or they affect other "
-"portions of the RE by repeating them or changing their meaning.  Much of "
-"this document is devoted to discussing various metacharacters and what they "
-"do."
-msgstr ""
+"`metacharacters`, and don't match themselves.  Instead, they signal that some "
+"out-of-the-ordinary thing should be matched, or they affect other portions of "
+"the RE by repeating them or changing their meaning.  Much of this document is "
+"devoted to discussing various metacharacters and what they do."
+msgstr ""
+"Il y a des exceptions à cette règle; certains caractères sont spéciaux :dfn:"
```

Comment:
"sont spéciaux metacharacters" does not look french at all.

Reply:
Done